ServerCore/Pages/Teams/Details.cshtml

```diff
@@ -27,15 +51,49 @@
 
 <h2>@Html.DisplayFor(model => model.Team.Name) Team</h2>
 
+@if (Model.Team.IsDisqualified)
+{
+    <div class="alert alert-danger" role="alert">
+        Your team has been disqualified from the event. You are welcome to
+        continue solving puzzles for fun, but your team will no longer appear
+        in the final standings and is not eligible for any awards or prizes
+        that the event may offer. If you have any questions, please contact 
+        <a href="@mailtoUrl">.
+    </div>
+}
+
 @if (Model.EventRole != ModelBases.EventRole.author)
 {
     <div>
-        <a asp-page="./Edit" asp-route-teamId="@Model.Team.ID">Edit team details</a> |
+        <a asp-page="./Edit" asp-route-teamId="@Model.Team.ID">Edit team details</a>
+        |
         @if (Model.EventRole == ModelBases.EventRole.admin)
         {
-            <text><a asp-page="/Events/Mailer" asp-route-group="Players" asp-route-teamId="@Model.Team.ID">Mail Players</a> |</text>
+            <text>
+            <a asp-page="/Events/Mailer" asp-route-group="Players" asp-route-teamId="@Model.Team.ID">Mail Players</a>
+            |
+            </text>
         }
         <a asp-page="./Delete" asp-route-teamId="@Model.Team.ID">Delete team</a>
+        |
+        @if (Model.EventRole == ModelBases.EventRole.admin)
+        {
+            @if (@Model.Team.IsDisqualified)
+            {
+                <a asp-page="./Disqualify" 
+                   asp-route-teamId="@Model.Team.ID">
+                    Requalify Team
```

Comment:
@(Model.Team.IsDisqualified ? "Requalify Team" : "Disqualify Team") - then you don't need to repeat the entire <a> tag.